packages/site-admin/.storybook/mock-registry/index.ts

```diff
@@ -26,7 +31,7 @@ const registerMockStore = ( registry, storeName, initialState ) => {
 /**
  * Creates a global mock registry for @wordpress/data
  */
-export const createMockRegistry = ( stores = {} ) => {
+export const createMockRegistry = ( stores: MockStore ): any => {
```

Comment:
Perhaps we could go with `import('@wordpress/data').WPDataRegistry` instead of `any`?

Reply:
Nice tip. Thanks! 🙇‍♂️